app/src/main/java/com/duckduckgo/app/tabs/model/TabDataRepository.kt

```diff
@@ -51,9 +51,7 @@ class TabDataRepository @Inject constructor(
     @AppCoroutineScope private val appCoroutineScope: CoroutineScope
 ) : TabRepository {
 
-    override val liveTabs: LiveData<List<TabEntity>> = tabsDao.liveTabs()
-
-    override val flowTabs: Flow<List<TabEntity>> = tabsDao.flowTabs()
+    override val flowTabs: Flow<List<TabEntity>> = tabsDao.flowTabs().distinctUntilChanged()
```

Comment:
I think livedata does not behave like flow in all cases. Is there anything special we'd need to test

Reply:
Nothing very specific apart from what's in the description.